recipe-server/normandy/recipes/api/v2/shield_identicon.py

```diff
@@ -0,0 +1,169 @@
+            if choice <= 0:
+                return option
+        raise Exception('No choices chosen.')
+
+    def emoji(self):
+        """Generates a random emoji."""
+        emojis = ["😄", "😃", "😀", "😊", "😉", "😍", "😘", "😚", "😗", "😙", "😜", "😝", "😛",
+                  "😳", "😁", "😔", "😌", "😒", "😞", "😣", "😢", "😂", "😭", "😪", "😥", "😰",
+                  "😅", "😓", "😨", "😱", "😠", "😡", "😤", "😖", "😆", "😋", "😷", "😎", "😴",
+                  "😵", "😲", "😟", "😦", "😧", "😈", "👿", "😮", "😬", "😐", "😯", "😶", "😇",
+                  "😏", "😑", "👼", "😺", "😻", "😽", "😼", "🙀", "😿", "😹", "😾", "👹", "👺",
+                  "🙈", "🙉", "🙊", "💀", "👽", "💩", "🔥", "✨", "🌟", "💫", "💥", "💦", "💧",
+                  "💤", "👂", "👀", "👃", "👅", "👄", "👍", "👎", "👌", "👊", "✊", "👋", "✋",
+                  "👐", "👆", "🙌", "🙏", "👏", "💪", "💃", "🎩", "👑", "👒", "👟", "👞", "👡",
+                  "👠", "👢", "💼", "👜", "👝", "👛", "👓", "🎀", "🌂", "💄", "💛", "💙", "💜",
+                  "💚", "💔", "💗", "💓", "💕", "💖", "💞", "💘", "💌", "💋", "💍", "💎", "👣",
+                  "🐶", "🐺", "🐱", "🐭", "🐹", "🐰", "🐸", "🐯", "🐨", "🐻", "🐷", "🐽", "🐮",
+                  "🐗", "🐵", "🐒", "🐴", "🐑", "🐘", "🐼", "🐧", "🐦", "🐤", "🐥", "🐣", "🐔",
+                  "🐍", "🐢", "🐛", "🐝", "🐜", "🐞", "🐌", "🐙", "🐚", "🐠", "🐟", "🐬", "🐳",
+                  "🐋", "🐄", "🐏", "🐀", "🐃", "🐅", "🐇", "🐉", "🐎", "🐐", "🐓", "🐕", "🐖",
+                  "🐁", "🐂", "🐲", "🐡", "🐊", "🐫", "🐪", "🐆", "🐈", "🐩", "🐾", "💐", "🌸",
+                  "🌷", "🍀", "🌹", "🌻", "🌺", "🍁", "🍃", "🍂", "🌿", "🌾", "🍄", "🌵", "🌴",
+                  "🌲", "🌳", "🌰", "🌱", "🌼", "🌐", "🌞", "🌝", "🌚", "🌜", "🌛", "🌙", "🌍",
+                  "🌎", "🌏", "⭐", "⛅", "⛄", "🌀", "💝", "🎒", "🎓", "🎏", "🎃", "👻", "🎄",
+                  "🎁", "🎋", "🎉", "🎈", "🔮", "🎥", "📷", "📹", "📼", "💿", "📀", "💽", "💾",
+                  "💻", "📱", "📞", "📟", "📠", "📡", "📺", "📻", "🔊", "🔔", "📢", "⏳", "⏰",
+                  "🔓", "🔒", "🔏", "🔐", "🔑", "🔎", "💡", "🔦", "🔆", "🔅", "🔌", "🔋", "🔍",
+                  "🛁", "🚿", "🚽", "🔧", "🔨", "🚪", "💣", "🔫", "🔪", "💊", "💉", "💰", "💸",
+                  "📨", "📬", "📌", "📎", "📕", "📓", "📚", "📖", "🔬", "🔭", "🎨", "🎬", "🎤",
+                  "🎵", "🎹", "🎻", "🎺", "🎷", "🎸", "👾", "🎮", "🃏", "🎲", "🎯", "🏈", "🏀",
+                  "⚽", "🎾", "🎱", "🏉", "🎳", "⛳", "🚴", "🏁", "🏇", "🏆", "🎿", "🏂", "🏄",
+                  "🎣", "🍵", "🍶", "🍼", "🍺", "🍻", "🍸", "🍹", "🍷", "🍴", "🍕", "🍔", "🍟",
+                  "🍗", "🍤", "🍞", "🍩", "🍮", "🍦", "🍨", "🍧", "🎂", "🍰", "🍪", "🍫", "🍬",
+                  "🍭", "🍯", "🍎", "🍏", "🍊", "🍋", "🍒", "🍇", "🍉", "🍓", "🍑", "🍌", "🍐",
+                  "🍍", "🍆", "🍅", "🌽", "🏠", "🏡", "⛵", "🚤", "🚣", "🚀", "🚁", "🚂", "🚎",
+                  "🚌", "🚍", "🚙", "🚘", "🚗", "🚕", "🚖", "🚛", "🚚", "🚨", "🚓", "🚔", "🚒",
+                  "🚑", "🚐", "🚲", "🚜", "💈", "🚦", "🚧", "🏮", "🎰", "🗿", "🎪", "🎭", "📍",
+                  "🚩", "💯"]
+        return self.choice(emojis)
+
+    def color(self):
+        """Generates a random color."""
+        return self.choice(self.colors)
+
+
+def pick_pair(genome, base_color):
```

Comment:
Since genome is the first argument, might as well make this a method on the class?

Reply:
I think that it might make more sense to make this a class method of the Color class instead.